pkgs/by-name/ht/httpy-cli/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, python3Packages
+, fetchPypi
+ }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "httpy-cli";
+  version = "1.1.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "httpy-cli";
+    hash = "sha256-uhF/jF4buHMDiXOuuqjskynioz4qVBevQhdcUbH+91Q=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    colorama
+    pygments
+    requests
+    urllib3
+  ];
+
```

Comment:
Please add a pythonImportsCheck

Can we execute the tests https://github.com/knid/httpy/tree/main/tests ?

Reply:
@hatch01 Have you addressed this comment?

Reply:
Oh my bad I forgot to push

Reply:
this was a mistake I added it !